ai-service/src/ai_service/ollama_client.py

```
@@ -0,0 +1,28 @@
+    Returns:
+        Model-generated response text.
+
+    Raises:
+        ConnectionError: If unable to connect to Ollama service.
+        KeyError: If the model is not available.
+        ResponseError: If response is invalid.
+    """
+
+    try:
+        response = ollama.chat(
+            model=model, messages=[{"role": "user", "content": prompt}]
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)